terralab/commands/auth_commands.py

```diff
@@ -2,7 +2,7 @@
 
 import click
 
-from logic import auth_logic
+from terralab.logic import auth_logic
```

Comment:
where these import changes related to stuff in this pr or from a previous pr?

Reply:
they were required to get the code working in the executable manner. i'm not sure why it wasn't an issue before and became an issue. this was inconsistently done previously.